src/pdm/cli/commands/run.py

```diff
@@ -279,8 +281,10 @@ def run_task(self, task: Task, args: Sequence[str] = (), opts: TaskOptions | Non
                 subargs = split[1:] + ([] if should_interpolate else args)
                 code = self.run(cmd, subargs, options, chdir=True)
                 if code != 0:
-                    return code
-            return code
+                    if not keep_going:
+                        return code
+                    composite_code += code
```

Comment:
What is the meaning of sum of the return code, if more than one processes return non-zero?

Reply:
Well, the expectation (mine expectation) is that composite will fail in case of a failure. Only change is that it will run all the "sobcommands", so non-zero code is needed. And so far I believe the exact value of the code does not matter that much, so I chose to do a sum. Anyway any value, even fixed should be appropriate, just the indication that something has failed (non-zero exit code) should remain.

In theory, if all the "subcommands" will fail with code 1 (or any other known value), then yes, I was also thinking that it could be useful information to preserve it somehow in the composite_code, therefore sum. Honestly I do not assume that this can have real use, but still there.